lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -216,6 +217,8 @@ namespace :foreman_tasks do
       tasks = ForemanTasks::Task.where("result != 'success'")
     end
 
+    days = ENV['days'].try(:to_i) || 60
```

Comment:
Maybe printing some informative message about the time limit, so that one has no expectations to get all by default